packages/@okta/vuepress-site/docs/api/resources/authn/index.md

```diff
@@ -4358,6 +4358,22 @@ curl -v -X POST \
 }
 ```
 
+##### Resend Push Notification
+
+Use the `resend` link to send another push notification if user doesn't receive notification due to timeout or rejection.
```

Comment:
How about:
Use the resend link to send another push notification if the user didn't receive the previous one due to timeout or error.

Reply:
updated text.